app/DoctrineMigrations/Version173.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version173 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('ALTER TABLE expense ADD bank_account_id INT DEFAULT NULL');
+        $this->addSql('ALTER TABLE expense ADD CONSTRAINT FK_2D3A8DA612CB990C FOREIGN KEY (bank_account_id) REFERENCES account (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('CREATE INDEX IDX_2D3A8DA612CB990C ON expense (bank_account_id)');
+
+        $this->addSql('ALTER TABLE gift ADD bank_account_id INT DEFAULT NULL');
```

Comment:
there is "ON DELETE" on  "expense", but not on "gift" table

Reply:
Fixed, forgot to re-run the migrations script after changing the schema. All bank account fields now set to null on deletion of the account. This will tie in with the error messaging next sprint